src/studio/src/designer/frontend/ux-editor/components/config/RuleComponent.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
-import * as uuid from 'uuid/v1'; // time
+import { v1 as uuidv1 } from 'uuid';
```

Comment:
What is the reason for using v1 here?

Reply:
I'm not sure. The default version given by uuid were version 4. For some reason the developer explicitly changed it to version 1 in this case. I haven't questioned why.

Reply:
Ok, then we'll leave it as is for now :)